README.md

```diff
@@ -1,19 +1,31 @@
 ## Lambda Handler for Probot
```

Comment:
How do you feel about tweaking this to **AWS Lambda extension for Probot** or something? I think it should mention AWS at least